mbq/metrics/__init__.py

```diff
@@ -24,20 +24,22 @@
 
 _constant_tags = []
 _initialized = False
-_namespace = None
+_service = None
+_env = None
 _statsd = datadog.DogStatsd(
     use_default_route=True,  # assumption: code is running in a container
 )
 
 
-def init(namespace=None, constant_tags=None):
-    global _constant_tags, _initialized, _namespace
+def init(service, env, constant_tags=None):
```

Comment:
`env` will be an `Environment` enum, right? So we'll need to make sure we pass the right thing when we make it a tag.

Reply:
oh I was thinking that env would be a string actually. Didn't think we wanted to make this library coupled with mbq.env. My thought was we pass in `env=ENV.long_name` when we initialize metrics. This is what we do for ranch and atomiq at least.